docs/conceptual/core/codec.md

```diff
@@ -120,19 +120,37 @@ SCALE Bytes:
 0x18040008000f00100017002a00
 ```
 
-### Tuples and Structures
+### Tuples
 
 A fixed-size series of values, each with a possibly different but predetermined
-and fixed type. This is simply the concatenation of each encoded value. For
-structures, the values are named, but that is irrelevant for the encoding (names
-are ignored - only order matters).
+and fixed type. This is simply the concatenation of each encoded value. 
 
 #### Example
 
 Tuple of compact unsigned integer and boolean:
 
 `(3, false)`: `0x0c00`
 
+### Data Structures
+
+For structures, the values are named, but that is irrelevant for the encoding (names are ignored - 
+only order matters). **All containers store elements consecutively. The order of the elements is not 
+fixed, depends on the container, and cannot be relied on at decoding.**
+
+The decoder is expected to verify that incoming bytes are valid, and silently interpret bytes so 
```

Comment:
What you mean by "verify that incoming bytes are valid"?

Reply:
I will just remove this confusing sentence regarding bytes validity